editor/gui/editor_file_dialog.cpp

```diff
@@ -1528,7 +1528,13 @@ void EditorFileDialog::_recent_selected(int p_idx) {
 }
 
 void EditorFileDialog::_go_up() {
-	dir_access->change_dir(get_current_dir().get_base_dir());
+	String cur_dir = get_current_dir();
+
+	if (cur_dir.ends_with("/")) {
+		cur_dir = cur_dir.left(-1);
+	}
```

Comment:
```suggestion
	String cur_dir = get_current_dir().trim_suffix("/");
```
This should behave the same.

Although I think both solutions would break `res://` (which can also be favorited). Probably needs extra check.

Reply:
should i run `Commit changes` via github? never done it before 😄 

![image](https://github.com/godotengine/godot/assets/41921395/e74e3970-905f-423b-9a46-14be83b908d3)

Reply:
It's better to apply the changes yourself. Also, as I mentioned, it needs some modification anyway.

Reply:
You can do that, but following that you would still have to pull these changes locally and then do a squash/fixup so that you end up with one single commit. So it may be quicker for you to do these changes locally instead of applying the suggestion :)